docs/tutorials/mutelist.md

```diff
@@ -108,23 +114,23 @@ If you want to mute failed findings only in specific regions, create a file with
             Resources:
               - "*"
 
-## Default AWS Mutelist
+### Default Mutelist
 For the AWS Provider, Prowler is executed with a Default AWS Mutelist with the AWS Resources that should be muted such as all resources created by AWS Control Tower when setting up a landing zone.
 You can see this Mutelist file in [`prowler/config/aws_mutelist.yaml`](https://github.com/prowler-cloud/prowler/blob/master/prowler/config/aws_allowlist.yaml).
 
-## Supported AWS Mutelist Locations
+### Supported Mutelist Locations
```

Comment:
I would not remove AWS, since this locations are only available for AWS Mutelists

Reply:
This is within a new AWS section.